### PR TITLE
Update Issue Template

### PR DESCRIPTION
**Pull request by PratyushJayachandran**
I noticed that the issue template can be simplified for getting website rankings from Similarweb.

Simplified the instructions to make it more straightforward.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/icon_request.yml [hunk 1]
```diff
@@ -17,8 +17,9 @@ body:
         you're requesting then leave a comment on it or add a reaction.
 
         > [!TIP]
-        > Similarweb is now trying to force users to log in in order to view statistics. You can bypass this by going directly to:
-        > `https://similarweb.com/website/google.com`, replacing `google.com` with the TLD you would like to get the stats on!
+        > In order to view website rankings on `similarweb.com``:
```
Comment (LitoMore) [inline .github/ISSUE_TEMPLATE/icon_request.yml:20]:
```suggestion
        > In order to view website rankings on `similarweb.com`:
```

Typo

**Review by LitoMore — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/icon_request.yml (hunk 1 above)
Comment (PratyushJayachandran) [reply .github/ISSUE_TEMPLATE/icon_request.yml]: Oops. Applied.

File: .github/ISSUE_TEMPLATE/icon_request.yml [hunk 2]
```diff
@@ -17,8 +17,9 @@ body:
         you're requesting then leave a comment on it or add a reaction.
 
         > [!TIP]
-        > Similarweb is now trying to force users to log in in order to view statistics. You can bypass this by going directly to:
-        > `https://similarweb.com/website/google.com`, replacing `google.com` with the TLD you would like to get the stats on!
+        > In order to view website rankings on `similarweb.com`:
+        > Go to Products at the navbar on top, click on Free tools and then select Analyze a website.
+        > Or go to `https://similarweb.com/website`, enter the website address and click on Analyze Traffic.
```
Comment (uncenter) [inline .github/ISSUE_TEMPLATE/icon_request.yml:22]:
```suggestion
        > Go to `https://similarweb.com/website`, enter the website address, and click on Analyze Traffic.
```

Is the first method necessary to include?

**Comment by PeterShaggyNoble** [comment]
I think what we currently have is perfectly fine; gets people to the information they're looking for as quickly as possible. If we _are_ going to change it, though, then I agree with @uncenter's comments above.

**Review by uncenter — APPROVED** [review]

**Review by adamrusted — APPROVED** [review]

**Pull request merged**